src/main/java/data/JsonConverter.java

```diff
@@ -55,7 +56,7 @@ public File createFile(String vendorID) {
   }
 
   public boolean updateFile(Vendor vendor) {
-    String jsonConfig = vendor.buildJsonConfig();
+    String jsonConfig = vendor.toJson();
```

Comment:
Is there a reason why you changed the method name to just toJson()? I personally think that buildJsonConfig() is a more descriptive name.

Reply:
I agree, changed it back :)